cmd/list.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the GPLv3, see LICENCE file for details.
+
+package cmd
+
+import (
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+)
+
+const listPlansDoc = `
+list-plans is to list plans
```

Comment:
list-plans lists all the plans owned by the specified user

Reply:
@mattyw it might be a group, not a user.. like.. canonical..